indico/modules/events/editing/operations.py

```diff
@@ -171,6 +171,7 @@ def replace_revision(revision, user, comment, files, tags, initial_state=None):
     revision.comment = comment
     revision.tags = tags
     revision.final_state = FinalRevisionState.replaced
+    revision.editor = user
```

Comment:
Just a small highlight on this change. Even though it helps this PR identifying the "author" of a revision action, I think it makes sense for replaced revisions to be identified by the *editor* that decided to replace them.

Reply:
the user is pretty much always the indico system account, but no harm in setting it. at least i think so.. ;)